gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/lru/NewLRUClockHand.java

```diff
@@ -244,6 +248,18 @@ public LRUClockNode getLRUEntry() {
         continue;
       }
 
+      // Checking whether this entry is outside lock ,
+      // so that we won;t attempt to evict an entry whose
+      // faultIn is in process
+      // TODO Remove SnappyStore check after 0.9 . Added this check to
+      // reduce regression cycles
+      if (snappyStore && (aNode.getValue() == null || aNode.testEvicted())) {
```

Comment:
as noted in the TODO please track removal of the "snappyStore" check post 0.9 release